SearchRotatedArray.cpp

```diff
@@ -0,0 +1,40 @@
+// Time complexity - It is O(logn) since we are applying binary search on the array. We are doing it in one pass and always reducing our search space by half.
+// Space complexity - O(n) since our search depends on the number of elements in the array? But I am not sure. Is my answer correct?
```

Comment:
Again here Space complexity is O(1) as we are using any extra space for searching an element.